diff --git a/bazel/repositories.bzl b/bazel/repositories.bzl
index f6b3466..108f9b5 100644
--- bazel/repositories.bzl
+++ bazel/repositories.bzl
@@ -5,7 +5,7 @@ load(":repository_locations.bzl", "REPOSITORY_LOCATIONS")
 load("@com_google_googleapis//:repository_rules.bzl", "switched_rules_by_language")
 
 # dict of {build recipe name: longform extension name,}
-PPC_SKIP_TARGETS = {"luajit": "envoy.filters.http.lua"}
+PPC_SKIP_TARGETS = {}
 NOBORINGSSL_SKIP_TARGETS = {
     # The lua filter depends on BoringSSL
     "lua": "envoy.filters.http.lua",
diff --git a/bazel/repository_locations.bzl b/bazel/repository_locations.bzl
index 802e939..86f8360 100644
--- bazel/repository_locations.bzl
+++ bazel/repository_locations.bzl
@@ -108,9 +108,9 @@ REPOSITORY_LOCATIONS = dict(
         urls = ["https://github.com/grpc/grpc/archive/v1.22.1.tar.gz"],
     ),
     com_github_luajit_luajit = dict(
-        sha256 = "409f7fe570d3c16558e594421c47bdd130238323c9d6fd6c83dedd2aaeb082a8",
+        sha256 = "3aa88b6ca158eb044fbaf928bd4838e1889626d2fb84f895f499d5746d464ae0",
         strip_prefix = "LuaJIT-2.1.0-beta3",
-        urls = ["https://github.com/LuaJIT/LuaJIT/archive/v2.1.0-beta3.tar.gz"],
+        urls = ["https://github.com/mgiessing/LuaJIT/archive/v2.1.0-beta3.tar.gz"],
     ),
     com_github_nanopb_nanopb = dict(
         sha256 = "cbc8fba028635d959033c9ba8d8186a713165e94a9de02a030a20b3e64866a04",
